consensus/vbft/service.go

```diff
@@ -304,14 +305,15 @@ func (self *Server) getChainConfig() (*vconfig.ChainConfig, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get peersinfo from leveldb: %s", err)
 	}
-	cfg, err := vconfig.GenesisChainConfig(config, peersinfo)
```

Comment:
self.chainStore.GetVbftConfigInfo() seems not related with chainstore.

change it to util function.